util/Setup/CertBuilder.cs

```diff
@@ -97,5 +97,19 @@ public void BuildForInstall()
                 Helpers.ShowBanner(_context, "WARNING", message, ConsoleColor.Yellow);
             }
         }
+
+        public void BuildForUpdater()
+        {
+            if (_context.Config.EnableKeyConnector && !File.Exists("/bitwarden/key-connector/bwkc.pfx"))
+            {
+                Directory.CreateDirectory("/bitwarden/key-connector/");
+                var KeyConnectorCertPassword = Helpers.GetValueFromEnvFile("key-connector",
```

Comment:
Minor syntax nit, local variables should be camelCase
```suggestion
                var keyConnectorCertPassword = Helpers.GetValueFromEnvFile("key-connector",
```